src/js/plugin-bridge/PluginSDK.js

```diff
@@ -297,14 +255,10 @@ const getSDK = function(pluginID, config) {
  * @param  {Object} config Plugin configuration
  * @param  {String} pluginType - One of 'internal' or 'external'.
```

Comment:
Should remove `pluginType` from the function documentation. Also, there are traces to `pluginType` that should be removed as well. There's a call to `bootstrapPlugin(pluginID, path, pluginsConfig[pluginID], pluginType);` in the `initialize` function.